website/docs/faqs.md

```diff
@@ -25,6 +25,13 @@ You can try use the parameters: `hostNetwork`. Usage is as follows:
 hostNetwork: true
 ```
 
+### Q: If install with kind reports `ERROR: failed to get cluster internal kubeconfig: command "docker exec --privileged kind-control-plane cat /etc/kubernetes/admin.conf" failed with error: exit status 1`, how can I fix it
```

Comment:
### Q: If you see an error message like this `xxxx` when installing Chaos Mesh with kind. How to fix it?

You can try the following command to fix it: